resources/ctia-default.properties

```diff
@@ -44,7 +44,6 @@ ctia.store.incident=es
 ctia.store.indicator=es
 ctia.store.judgement=es
 ctia.store.relationship=es
-ctia.store.verdict=es
```

Comment:
We'll need a tracking PR for tenzin to update the ctia.properties file there, as well.